scripts/setDefaultIndex.sh

```diff
@@ -27,9 +27,8 @@ if [ ! -e $KIBANA_LOG_FILE ]; then
 fi
 
 # Create the index pattern and specify TimeUpdated as the default time parameter
-curl -g -i -XHEAD localhost:9200/.kibana/index-pattern/[network_]YYYY_MM_DD | grep -q "404"
-index_pattern_exists=$?
-if [ "$index_pattern_exists" -eq "0"  ]; then
+index_pattern_exists=`curl -g -i -XHEAD localhost:9200/.kibana/index-pattern/[network_]YYYY_MM_DD | grep "200"`
+if [ $? -ne 0  ]; then
```

Comment:
I think you want to log the index_pattern_exists if the curl command does not return 200. It will be interesting to see if there are times when 404 is not returned in that case.